Description: Fixes paste in rxvt-unicode 9.11
Bugs-Debian: http://bugs.debian.org/631264
Origin: Upstream
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Updated: 2011-06-22 
 
Index: pkg-urxvt/src/rxvttoolkit.C
===================================================================
--- pkg-urxvt.orig/src/rxvttoolkit.C	2011-06-22 07:30:33.027019240 -0400
+++ pkg-urxvt/src/rxvttoolkit.C	2011-06-22 07:30:39.739052529 -0400
@@ -1183,8 +1183,7 @@
 
       case SelectionNotify:
         if (selection_wait == Sel_normal
-            && xev.xselection.time == request_time
-            && xev.xselection.property == request_prop)
+            && xev.xselection.time == request_time)
           {
             timer_ev.stop ();
             handle_selection (xev.xselection.requestor, xev.xselection.property, true);
