Index: rxvt-unicode-8.9/src/rxvttoolkit.C
===================================================================
--- rxvt-unicode-8.9.orig/src/rxvttoolkit.C	2007-12-24 03:02:12.000000000 -0500
+++ rxvt-unicode-8.9/src/rxvttoolkit.C	2007-12-26 20:07:07.000000000 -0500
@@ -321,13 +321,15 @@
   // 6. System wide per application default file.
 
   /* Add in $XAPPLRESDIR/Rxvt only; not bothering with XUSERFILESEARCHPATH */
-  if ((xe = getenv ("XAPPLRESDIR")))
+  if (!(xe = getenv ("XAPPLRESDIR")))
     {
-      snprintf (fname, sizeof (fname), "%s/%s", xe, RESCLASS);
-
-      if ((rdb1 = XrmGetFileDatabase (fname)))
-        XrmMergeDatabases (rdb1, &database);
+      // Debian-specific; see #348697 and Policy
+      xe = "/etc/X11/app-defaults";
     }
+  snprintf (fname, sizeof (fname), "%s/%s", xe, RESCLASS);
+
+  if ((rdb1 = XrmGetFileDatabase (fname)))
+    XrmMergeDatabases (rdb1, &database);
 
   // 5. User's per application default file.
   // none
