Index: rxvt-unicode-7.8/src/rxvttoolkit.C
===================================================================
--- rxvt-unicode-7.8.orig/src/rxvttoolkit.C	2006-07-24 14:01:30.000000000 -0400
+++ rxvt-unicode-7.8/src/rxvttoolkit.C	2006-07-24 14:01:34.000000000 -0400
@@ -320,13 +320,15 @@
   // 6. System wide per application default file.
 
   /* Add in $XAPPLRESDIR/Rxvt only; not bothering with XUSERFILESEARCHPATH */
-  if ((xe = (char *)getenv ("XAPPLRESDIR")))
+  if (!(xe = (char *)getenv ("XAPPLRESDIR")))
     {
-      snprintf (fname, sizeof (fname), "%s/%s", xe, RESCLASS);
-
-      if ((rdb1 = XrmGetFileDatabase (fname)))
-        XrmMergeDatabases (rdb1, &database);
+      // Debian-specific; see #348697 and Policy
+      xe = "/etc/X11/app-defaults";
     }
+  snprintf (fname, sizeof (fname), "%s/%s", xe, RESCLASS);
+
+  if ((rdb1 = XrmGetFileDatabase (fname)))
+    XrmMergeDatabases (rdb1, &database);
 
   // 5. User's per application default file.
   // none
