Index: rxvt-unicode-7.7/doc/Makefile.in
===================================================================
--- rxvt-unicode-7.7.orig/doc/Makefile.in	2006-03-12 12:59:00.000000000 -0500
+++ rxvt-unicode-7.7/doc/Makefile.in	2006-03-12 13:00:10.000000000 -0500
@@ -51,13 +51,13 @@
 	./podtbl <$< >$@
 
 %.1.man: %.1.tbl
-	$(SED) $(SEDREPLACE) <$< | pod2man -n rxvt -r"$(VERSION)" -q\" -s1 -c "RXVT-UNICODE" >$@
+	$(SED) $(SEDREPLACE) <$< | pod2man -n urxvt -r"$(VERSION)" -q\" -s1 -c "RXVT-UNICODE" >$@
 
 %.3.man: %.3.tbl
-	$(SED) $(SEDREPLACE) <$< | pod2man -n rxvt -r"$(VERSION)" -q\" -s3 -c "RXVT-UNICODE" >$@
+	$(SED) $(SEDREPLACE) <$< | pod2man -n urxvt -r"$(VERSION)" -q\" -s3 -c "RXVT-UNICODE" >$@
 
 %.7.man: %.7.tbl
-	$(SED) $(SEDREPLACE) <$< | pod2man -n rxvt -r"$(VERSION)" -q\" -s7 -c "RXVT-UNICODE" >$@
+	$(SED) $(SEDREPLACE) <$< | pod2man -n urxvt -r"$(VERSION)" -q\" -s7 -c "RXVT-UNICODE" >$@
 
 %.html: %.tbl
 	$(SED) $(SEDREPLACE) <$< | pod2html >$@
