Index: rxvt-unicode-8.3/src/urxvt.pm
===================================================================
--- rxvt-unicode-8.3.orig/src/urxvt.pm	2007-08-07 14:24:41.000000000 -0400
+++ rxvt-unicode-8.3/src/urxvt.pm	2007-08-07 14:47:31.000000000 -0400
@@ -103,7 +103,7 @@
 onto C<@{ $term->{option_popup_hook} }>, which gets called whenever the
 popup is being displayed.
 
-It's sole argument is the popup menu, which can be modified.  It should
+Its sole argument is the popup menu, which can be modified.  It should
 either return nothing or a string, the initial boolean value and a code
 reference. The string will be used as button text and the code reference
 will be called when the toggle changes, with the new boolean value as
@@ -126,7 +126,7 @@
 onto C<@{ $term->{selection_popup_hook} }>, which gets called whenever the
 popup is being displayed.
 
-It's sole argument is the popup menu, which can be modified. The selection
+Its sole argument is the popup menu, which can be modified. The selection
 is in C<$_>, which can be used to decide whether to add something or not.
 It should either return nothing or a string and a code reference. The
 string will be used as button text and the code reference will be called
@@ -137,7 +137,7 @@
 C<a>s:
 
    push @{ $self->{term}{selection_popup_hook} }, sub {
-      /a/ ? ("a to be" => sub { s/a/b/g }
+      /a/ ? ("a to b" => sub { s/a/b/g }
           : ()
    };
 
@@ -278,7 +278,7 @@
 
 This (experimental) perl extension implements OnTheSpot editing. It does
 not work perfectly, and some input methods don't seem to work well with
-OnTheSpot editing in general, but it seems to work at leats for SCIM and
+OnTheSpot editing in general, but it seems to work at least for SCIM and
 kinput2.
 
 You enable it by specifying this extension and a preedit style of
@@ -296,7 +296,7 @@
 
 Initially, the window will not be shown when using this extension.
 
-This is useful if you need a single terminal thats not using any desktop
+This is useful if you need a single terminal that's not using any desktop
 space most of the time but is quickly available at the press of a key.
 
 The accelerator key is grabbed regardless of any modifiers, so this
@@ -331,7 +331,7 @@
 =item remote-clipboard
 
 Somewhat of a misnomer, this extension adds two menu entries to the
-selection popup that allows one ti run external commands to store the
+selection popup that allows one to run external commands to store the
 selection somewhere and fetch it again.
 
 We use it to implement a "distributed selection mechanism", which just
