Index: rxvt-unicode-8.7/src/perl/selection
===================================================================
--- rxvt-unicode-8.7.orig/src/perl/selection	2007-12-12 23:46:50.000000000 -0500
+++ rxvt-unicode-8.7/src/perl/selection	2007-12-13 00:01:03.000000000 -0500
@@ -12,11 +12,13 @@
 sub on_init {
    my ($self) = @_;
 
+   my $cutchars = '\\`"\'&()*,;<=>?@[]{|}';
    if (defined (my $res = $self->resource ("cutchars"))) {
-      $res = $self->locale_decode ($res);
-      push @{ $self->{patterns} }, qr{\G [\Q$res\E[:space:]]* ([^\Q$res\E[:space:]]+) }x;
+      $cutchars = $self->locale_decode ($res);
    }
 
+   push @{ $self->{patterns} }, qr{\G [\Q$cutchars\E[:space:]]* ([^\Q$cutchars\E[:space:]]+) }x;
+
    for (my $idx = 0; defined (my $res = $self->x_resource ("selection.pattern-$idx")); $idx++) {
       $res = $self->locale_decode ($res);
       utf8::encode $res;
