Index: rxvt-unicode-8.3/src/perl/selection-popup
===================================================================
--- rxvt-unicode-8.3.orig/src/perl/selection-popup	2007-08-07 14:02:03.000000000 -0400
+++ rxvt-unicode-8.3/src/perl/selection-popup	2007-08-07 14:02:36.000000000 -0400
@@ -14,7 +14,7 @@
 sub on_start {
    my ($self) = @_;
 
-   $self->{browser} = $self->x_resource ("urlLauncher") || "x-www-browser";
+   $self->{browser} = $self->x_resource ("urlLauncher") || "sensible-browser";
 
    $self->grab_button (3, urxvt::ControlMask);
 
