Index: rxvt-unicode-9.02/src/perl/tabbed
===================================================================
--- rxvt-unicode-9.02.orig/src/perl/tabbed	2008-01-29 23:06:00.000000000 -0500
+++ rxvt-unicode-9.02/src/perl/tabbed	2008-01-29 23:06:44.000000000 -0500
@@ -102,7 +102,7 @@
    my %our_props = map +($_ => undef), $self->XListProperties ($self->parent);
 
    delete $our_props{$self->XInternAtom ($_)}
-      for qw(WM_STATE WM_ICON_SIZE
+      for qw(WM_STATE WM_ICON_SIZE _NET_WM_STATE _NET_WM_DESKTOP
              _NET_WM_VISIBLE_NAME _NET_WM_VISIBLE_ICON_NAME _NET_WM_ICON_GEOMETRY _NET_FRAME_EXTENTS);
 
    for my $atom ($tab->XListProperties ($tab->parent)) {
